tests/plugins/plugin.text_selection.test.js

```diff
@@ -0,0 +1,102 @@
+import sinon from 'sinon';
```

Comment:
Add tests for:

- newline added at end of paragraph
- no space added at end of paragraph
- coordinates containing 5 letters (copy from the Japanese book we were testing)
- does not create new SVG if already there
- works if paragraph has exactly one word
- works if paragraph has more than one word

Reply:
paragraph element call jquery .outer(), or non-jquery outerHTML (something like that) and check if it has `\n` in it.

Or, you can do something `$('text')[0].nextSiblingNode` (or something like that)

Reply:
Or actually, you've met the codecov limits, so lets not bother :P